src/utils/fill-grid/index.ts

```diff
@@ -10,38 +10,39 @@ import {
 
 const numbers: NUMBERS[] = [1, 2, 3, 4, 5, 6, 7, 8, 9]
 
-/**
- * A backtracking/recursive function to check all the possible combinations of numbers a solution is found
- * @param grid  9X9 Sudoku Grid
- */
 function fillGrid(grid: GRID) {
   let row = 0
   let col = 0
 
   for (let i = 0; i < 81; i++) {
     row = Math.floor(i / 9)
     col = i % 9
-
+/* eslint-disable */
+/**
+ * eslint prefer-const not let 
+ */

```

Comment:
We can make this better by simply disabling the rule for the next line. 

`// eslint-disable-next-line rule-name`

This way you don't have to disable eslint for the whole block like you are doing right now. Just make sure to check that you have the name of the rule right. There is never space between rules so it is likely named `prefer-const-not-let` or simply `prefer-const`

Reply:
👍🏼